apps/frontend/src/hooks/instrument/useInstrumentsMinimalData.ts

```diff
@@ -0,0 +1,78 @@
+import {
```

Comment:
This looks very similar to `useInstrumentsData.ts`. Should duplicated code be reduced or would it be preferable to keep them entirely separate?

Reply:
I tried to, but I could not for the life of me get the typing to work in some way you would have something along the lines of `InstrumentDataMinimal | InstrumentData` which I could not get to type check in the places that needed `InstrumentData`. Happy to be proven wrong, but I could not make it work when I tryed.

Reply:
![image](https://github.com/user-attachments/assets/bc0f18ca-6412-4ecd-b922-236514ad2bf4)
I'll take that pay raise now, please.